rpcs3/rpcs3qt/pad_settings_dialog.cpp

```diff
@@ -530,7 +530,7 @@ void pad_settings_dialog::InitButtons()
 
 		while (thread_ctrl::state() != thread_state::aborting)
 		{
-			thread_ctrl::wait_for(10000);
```

Comment:
Maybe what you need to do, is sleeping until X time, then increasing X by 10000 each time. Then it would account for the time spent in the code.

Reply:
This already is the input thread. So it would change nothing.